src/SOS/Strike/platformspecific.h

```diff
@@ -51,6 +51,11 @@ struct DT_UNICODE_STRING
 #define IMAGE_FILE_MACHINE_ARM64             0xAA64  // ARM64 Little-Endian
 #endif
 
+#ifndef IMAGE_FILE_MACHINE_ARM64X
+#define IMAGE_FILE_MACHINE_ARM64EC 0xA641
```

Comment:
Should it go under its own ifndef condition?

Reply:
This patch needs some work, but basically if one isn't defined it should define both - this just means an old win SDK